### PR TITLE
Feat(eos_cli_config_gen): Support cos in policy-map and service-policy in qos profile

### PR DESCRIPTION
**Pull request by tgodaA**
## Change Summary

Added knobs to set **cos** and its value in **policy_maps**, and qos **service_policy** in **qos_profiles**.

## Component(s) name

`arista.avd.eos_cli_config_gen`

## Proposed changes

Added the new knobs:

1) cos
```
policy_maps:
  qos:
    < policy-map name >:
      classes:
        < class name >:
          set:
            cos: < cos_value >
```

2) service_policy
```
qos_profiles:
  < profile_name >:
    service_policy:
      type:
        qos_input: < policy_map_name >
```

## How to test

molecule

### User Checklist

- N/A

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly.
- [x] I have updated [molecule CI](https://github.com/aristanetworks/ansible-avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

**Review by carlbuchmann — APPROVED** [review]
LGMT!

File: ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/policy-maps.md [hunk 1]
```diff
@@ -98,7 +98,7 @@ interface Management1
 **PM_REPLICATION_LD**
 
 | class | Set | Value |
```
Comment (emilarista) [inline ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/policy-maps.md:100]: Is it on purpose to have non-capitalized class?

Comment (tgodaA) [reply ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/policy-maps.md:100]: Not really, it was old file. I can modify them.

Comment (emilarista) [reply ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/policy-maps.md:100]: True, well maybe that can be part of a separate PR so we keep feature PRs small...

**Review by emilarista — APPROVED** [review]
LGTM!

**Pull request merged**